doc/manual/step-by-step.rst

```diff
@@ -1,9 +1,14 @@
-====================
-Step by step process
-====================
+============================
+Step by step process (Dutch)
+============================
 
 .. note:: This section is in Dutch and follows the same steps as the :ref:`demo`.
 
+Hieronder staat een typische workflow door het vernietigingsproces zoals dat
+mogelijk wordt gemaakt met deze applicatie. Het is mogelijk om het proces iets
+anders in te richten met behoudt van dezelfde functionaliteiten. Deze globale
+functionaliteiten staat beschreven in :ref:`destruction`.
```

Comment:
Is it intended to have "functionaliteiten" plural and "staat" singular?

Reply:
*snif* Your Dutch is better than mine